crud-cert-auth/src/main/webapp/WEB-INF/web.xml

```diff
@@ -1,11 +1,6 @@
 <web-app version="3.0" xmlns="http://java.sun.com/xml/ns/javaee"
 xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
 xsi:schemaLocation="http://java.sun.com/xml/ns/javaee http://java.sun.com/xml/ns/javaee/web-app_3_0.xsd">
-
-  <servlet-mapping>
-     <servlet-name>com.redhat.lightblue.rest.crud.RestApplication</servlet-name>
-     <url-pattern>/*</url-pattern>
-  </servlet-mapping>
```

Comment:
I removed this because not all servlet containers support this JAX-RS shorthand. I replaced it with an annotation on the class, which should be equivalent for JBoss.